src/test/java/htsjdk/variant/vcf/VCFHeaderUnitTest.java

```diff
@@ -362,6 +362,13 @@ public void testVCFHeaderSerialization() throws Exception {
         Assert.assertEquals(deserializedHeader.toString(), originalHeader.toString(), "String representation of header not the same before/after serialization");
     }
 
+    @Test(expectedExceptions = TribbleException.class)
+    public void testContigsAbsenceInHeader() {
+        String vcf = "src/test/resources/htsjdk/variant/HiSeq.10000.vcf";
```

Comment:
I think it would be a good idea to create a separate, small test file for this case, named something like "noContigLines.vcf", that is otherwise valid, rather than relying on the fact that an existing test file happens to satisfy the criteria.